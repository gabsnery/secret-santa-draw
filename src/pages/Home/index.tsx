import Footer from "../../components/footer";
import Form from "../../components/form";
import Participants from "../../components/participants";

function App() {
  return (
    <>
      <Form/>
      <Participants/>
      <Footer/>
    </>
  );
}
export default App;