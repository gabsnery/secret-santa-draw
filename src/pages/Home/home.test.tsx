import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Home from ".";
const mockNavigation=jest.fn();

jest.mock('react-router-dom',()=>{
    return {useNavigate:()=>mockNavigation}
})
describe('home page', () => {
    test("shoudl render", () => {
        const { container } = render(<RecoilRoot>
            <Home />
        </RecoilRoot>)
        expect(container).toMatchSnapshot()
    })
})