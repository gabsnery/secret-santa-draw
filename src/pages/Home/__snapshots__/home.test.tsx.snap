// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page shoudl render 1`] = `
<div>
  <form>
    <input
      placeholder="Participant\`s name"
      type="text"
      value=""
    />
    <button
      disabled=""
      type="submit"
    >
      add
    </button>
    
  </form>
  <ul />
  <button
    disabled=""
  >
    Draw
  </button>
</div>
`;
